[12.0][IMP] cooperator_website: backport changes from migration to 14.0

huguesdk opened the pull request:
## changes

*   improve web form placeholder values and error messages.
*   some code cleanup.
*   improve `fr` translations.

huguesdk committed:
[IMP] backport changes from migration to 14.0

co-authored-by: Victor Champonnois <victor@coopiteasy.be>

victor-champonnois reviewed (approved):
LGTM. The error messagesare much better !

robinkeunen committed:
[REM] cooperator: jquery.inputmask.bundle.js

inputmask was used in version 9 to force the formatting of
birthdate in the subscription request form. It is no longer
useful and allows to enable eslint.

huguesdk commented:
@cathLemb or @polchampion: could you please validate this functionally? it is deployed on the test servers (on all dbs, and should work on a new db also, of course). all changes are listed in the description.

robinkeunen committed:
Merge pull request #383 from coopiteasy/12.0-remove-input-mask

[REM] cooperator: jquery.inputmask.bundle.js

huguesdk committed:
[IMP] use normal subscription.request.create()

… instead of the deprecated .create_comp_sub_req().

huguesdk committed:
[IMP] convert method name to snake case

rename .preRenderThanks() to .pre_render_thanks().

huguesdk commented:
/ocabot merge minor
